packages/eas-cli/src/commands/build/index.ts

```diff
@@ -93,6 +112,17 @@ export default class Build extends EasCommand {
       default: false,
       description: 'Clear cache before the build',
     }),
+    submit: flags.boolean({
```

Comment:
maybe `--auto-submit`

Reply:
@brentvatne what do you think?

Reply:
i talked with @ide and @EvanBacon about this a bit, and i think @wkozyra95's suggestion is pretty reasonable. "auto submit is clearer at a glance to me. makes it clear that another action is taking place" - @ide.

as i said in the original doc, we could bikeshed on this for days. i don't think any of the options feel like they fit perfectly but this seems like the best for now, we can always revisit if we have some other insight.

so, the flags could be: `--auto-submit` and `--auto-submit-with-profile`. if we use the names, only one of the flags should be supplied at any given time: `eas build -p ios --auto-submit --auto-submit-with-profile beta` wouldn't make sense (we can ignore the `--auto-submit` and give the profile precedence here). 

```
eas build -p ios --profile abcd --auto-submit-with-profile beta 
eas build -p ios --profile abcd --auto-submit
```

my main concern is how long the `--auto-submit-with-profile` name is. but let's roll with it to unblock.

Reply:
TIL https://en.wiktionary.org/wiki/bikeshedding Thanks @brentvatne :D

Reply:
> if we use the names, only one of the flags should be supplied at any given time
This is exactly how I implemented it. See https://github.com/expo/eas-cli/pull/603/files#diff-cd2e00f4d286a4714f8736e14c635f4c3b98c402367cd71a83e4cd743034bd79R119